data/fields/substation.json

```diff
@@ -1,5 +1,18 @@
 {
     "key": "substation",
     "type": "typeCombo",
-    "label": "Type"
+    "label": "Type",
+    "strings": {
+        "options": {
+          "transmission": "HV transmission",
+          "distribution": "HV/MV distribution",
+          "minor_distribution": "LV distribution",
```

Comment:
It took me a little while to figure out what these nonstandard initialisms stand for. Can we spell them out?